docs/en/test_and_ci/integration_testing_mavsdk.md

```diff
@@ -7,6 +7,10 @@ In future we plan to generalise them for any platform/hardware.
 
 The instructions below explain how to setup and run the tests locally.
 
+:::note
+This is the recommended integration test framework for PX4.
+:::
+
 ## Prerequisites
 
 ### Setup Developer Environment
```

Comment:
@julianoes Just FYI, we need to get MAVSDK tests working with Gazebo rather than Gazebo classic (as discussed). This is one place where we would need to update the docs once that happens.

Reply:
I know. On it.